packages/node_modules/@webex/internal-plugin-conversation/README.md

```diff
@@ -2,7 +2,7 @@
 
 [![standard-readme compliant](https://img.shields.io/badge/readme%20style-standard-brightgreen.svg?style=flat-square)](https://github.com/RichardLitt/standard-readme)
 
-> Plugin for the Conversation service
+> Plugin for the Internal Conversation service
```

Comment:
Should we update all plugins that hit internal services to do this if we haven't already?

Reply:
This commit will probably be reverted, it was done just to test the changed packages script. Not meant to be the de-facto way of documenting internal plugins.